app/src/main/java/rec/games/pokemon/teambuilder/model/PokeAPIUtils.java

```diff
@@ -93,6 +93,16 @@ public static String getArtworkUrl(int id)
 			.appendEncodedPath(Integer.toString(id) + POKE_API_SPRITE_FILE_TYPE).build().toString();
 	}
 
+	public static String fixStaticAPIUrl(String url)
+	{
+		//the url.substring(1) is to avoid "//" from appearing in the url
+		//while that is a valid Uri, don't want to risk a server not processing it correctly
+		return Uri.parse(STATIC_POKE_API_BASE_URL).buildUpon()
+			.appendEncodedPath(url.substring(1))
+			.appendPath(STATIC_POKE_API_FILE_NAME)
```

Comment:
You only want to append `index.json` to urls ending in '/', right?

Reply:
For some odd reason, all of the equivalent endpoints in PokeAPI/api-data end with index.json. So they all need it

Reply:
Oh, everything is `index.json`? Ew. Okay.